src/components/admin/NewFormBtn.tsx

```diff
@@ -0,0 +1,71 @@
+import { useNavigate } from "react-router-dom";
+import { ulid } from "ulid";
+import useAdminForm from "./hooks/useAdminForm";
+import styles from "./newFormBtn.module.css";
+
+const NewFormBtn = () => {
+  const navigate = useNavigate();
+  const formId = ulid();
+  const { lists } = useAdminForm();
+
+  const onCopyClick = ({ key }: { key: string }) => {
+    const urlToCopy = `/user/forms/${key}`;
+    navigator.clipboard
+      .writeText(urlToCopy)
+      .then(() => {
+        alert("URL이 클립보드에 복사되었습니다!");
+      })
+      .catch((err) => {
+        console.error("복사에 실패했습니다:", err);
+      });
+  };
+
+  // console.log(lists);
+
+  return (
+    <>
+      <ul className={styles.wrapper}>
+        {lists &&
+          lists?.map((list) => (
```

Comment:
P5 ask;

아래처럼 되게도 할 수 있을 것 같은데! 어떻게 할 수 있을까요!? 

```suggestion
        {lists.map((list) => (
```

Reply:
const [lists, setLists] = useState<FormsLists[]>([]);
요렇게 초깃값을 넣고, 
  if (lists.length === 0) {
    return <div>Loading...</div>; 
  }
  이런식으로 
  lists.length가 0일경우 Loading을 보여주면 될것 같네요!